src/rbac/main.py

```
@@ -7,17 +7,17 @@
 logging.basicConfig(
     level=logging.INFO,
     format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
 )
 
 app = FastAPI(lifespan=lifespan)
 app.add_middleware(RequestContextMiddleware)
-
-
+# TODO app.handler가 등록이 안돼서 임의적으로 실행시키려고 임포트하였는데 다른 방법을 찾아봐야 할거 같음.
+import rbac.application.exception.controller_exception_handler as exception_handler
+exception_handler
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)